gateway/src/apicast/oauth/oidc.lua

```diff
@@ -41,11 +41,33 @@ function _M.new(service)
     keys = oidc.keys or empty,
     clock = ngx_now,
     alg_whitelist = util.to_hash(config.id_token_signing_alg_values_supported),
+    -- https://tools.ietf.org/html/rfc7523#section-3
```

Comment:
We don't need to specify anything about 7?
```
   7.   The JWT MAY contain a "jti" (JWT ID) claim that provides a
        unique identifier for the token.  The authorization server MAY
        ensure that JWTs are not replayed by maintaining the set of used
        "jti" values for the length of time for which the JWT would be
        considered valid based on the applicable "exp" instant.
```

Reply:
We don't do anything about it, so I don't think so.